zio-http/shared/src/main/scala/zio/http/codec/HttpContentCodec.scala

```diff
@@ -13,25 +13,31 @@ import zio.http.Header.Accept.MediaTypeWithQFactor
 import zio.http._
 import zio.http.internal.HeaderOps
 
+final case class BinaryCodecWithSchema[A](codec: BinaryCodec[A], schema: Schema[A])
```

Comment:
This should be in `BinaryCodecWithSchema.scala`

Reply:
done